fe/fe-core/src/main/java/com/starrocks/load/streamload/StreamLoadTask.java

```diff
@@ -1292,6 +1291,10 @@ public String getStateName() {
         return state.name();
     }
 
+    public TUniqueId getTUniqueId() {
```

Comment:
interface seems unnecessary, in the unit test, the unique id is set manually, no need to get uniqueid from the task.

Reply:
sorry it's a mistake

Reply:
> interface seems unnecessary, in the unit test, the unique id is set manually, no need to get uniqueid from the task.

This method is used by unit test

Reply:
I mean in unit test, the unique id is set in the testbody, why not directly use the value instead of get the unique id again from the stream task?